crates/minitrace-datadog/src/lib.rs

```diff
@@ -9,6 +9,9 @@ use std::net::SocketAddr;
 
 pub fn encode(
     service_name: &str,
+    r#type: &str,
```

Comment:
Could we pick a normal name for it?

Reply:
Sure. `trace_type`? Do you prefer using serde rename on the struct or keep that `r#type`?

Reply:
I prefer to use serde rename